layers/meta-resin-edison/recipes-core/images/resin-image.bbappend

```diff
@@ -59,6 +59,89 @@ populate_append_edison() {
     install -m 0644 ${DEPLOY_DIR_IMAGE}/${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin ${DEST}/vmlinuz
 }
 
+# XXX
+# This function is copied from poky/meta/classes/bootimg.bbclass in
+# order to support resinhup on the Edison. Even though it's used for a
+# variety of purposes in Poky, it only generates the boot partition on our
+# version of the Edison HostOS. resinhup requires this partition to be
+# 40MB in size. The problem with generating this partition with this
```

Comment:
Does it require it to be "at least 40MB", "no more than 40MB" or "exactly 40MB"?